extensions/usAnnotation/src/getCommandsModule.ts

```diff
@@ -0,0 +1,361 @@
+import { UltrasoundAnnotationTool, Enums as csToolsEnums } from '@cornerstonejs/tools';
+import { Types as OhifTypes } from '@ohif/core';
+import { setShowPercentage } from './getCustomizationModule';
+import { eventTarget, triggerEvent, utilities } from '@cornerstonejs/core';
+
+const { transformWorldToIndex } = utilities;
+
+/**
+ * Retrieves the DICOM instance associated with a specific imageId
+ * @param services - The OHIF services object
+ * @param imageId - The image ID to find the instance for
+ * @returns The DICOM instance object or undefined if not found
+ */
+function getInstanceByImageId(services: AppTypes.Services, imageId: string) {
+  const activeDisplaySets = services.displaySetService.getActiveDisplaySets();
+  const displaySet = activeDisplaySets.find(displaySet => displaySet?.imageIds?.includes(imageId));
+  return displaySet?.instance;
+}
+
+/**
+ * Creates and returns the commands module for ultrasound annotation
+ * @param params - Extension parameters including servicesManager and commandsManager
+ * @returns The commands module with actions and definitions
+ */
+function commandsModule({
+  servicesManager,
+  commandsManager,
+}: OhifTypes.Extensions.ExtensionParams): OhifTypes.Extensions.CommandsModule {
+  const { viewportGridService, toolGroupService, cornerstoneViewportService } =
+    servicesManager.services as AppTypes.Services;
+
+  const actions = {
+    /**
+     * Switches the active ultrasound annotation type
+     * @param options - Object containing the annotationType to switch to
+     */
+    switchUSAnnotation: ({ annotationType }) => {
+      const activeViewportId = viewportGridService.getActiveViewportId();
+      const toolGroup = toolGroupService.getToolGroupForViewport(activeViewportId);
+      if (!toolGroup) {
+        return;
+      }
+      const usAnnotation = toolGroup.getToolInstance(UltrasoundAnnotationTool.toolName);
+      if (usAnnotation) {
+        usAnnotation.setActiveAnnotationType(annotationType);
+      }
+    },
+    /**
+     * Convenience method to switch to pleura line annotation type
+     */
+    switchUSAnnotationToPleuraLine: () => {
+      actions.switchUSAnnotation({
+        annotationType: UltrasoundAnnotationTool.USAnnotationType.PLEURA,
+      });
+    },
+    /**
+     * Convenience method to switch to B-line annotation type
+     */
+    switchUSAnnotationToBLine: () => {
+      actions.switchUSAnnotation({
+        annotationType: UltrasoundAnnotationTool.USAnnotationType.BLINE,
+      });
+    },
+    /**
+     * Deletes the last annotation of the specified type
+     * @param options - Object containing the annotationType to delete
+     */
+    deleteLastAnnotation: ({ annotationType }) => {
```

Comment:
This command is specific to this tool and extension. The command name should reflect this, so users don't mistake it for a general "delete last annotation" function.